src/main/java/com/github/nicholasmoser/gnt4/seq/opcodes/UnknownOpcode.java

```diff
@@ -37,6 +40,20 @@ public static UnknownOpcode of(int bytesLength, ByteStream bs, String info) thro
     return new UnknownOpcode(bs.offset() - bytesLength, bytes, info);
   }
 
+  public static byte[] of(String opcode, String operands) {
+    String[] op = operands.split(",");
+    int len = (op.length-1)*4;
+    ByteBuffer bytes = ByteBuffer.allocate(len);
+    bytes.putShort(Short.parseShort(opcode,16));
+    bytes.put(Byte.decode(op[0]));
+    bytes.put(Byte.decode(op[1]));
```

Comment:
May not want to use `Byte.decode(...)` here for a very weird reason. If you pass in 0x80 - 0xFF it actually fails. This is because bytes in Java are between -128 and 127, and Java thinks that 128 (0x80) and higher are out of range.

Here's a unit test you can use as proof, taking from Iruka's 0010.seq file:

```java
    /**
     * Test an opcode from Iruka's 0010.seq with an operand larger than 0x80.
     */
    @Test
    public void testUnknownOpcodeOfLargeBytes() {
        // 0678C | op_26DA {26DA01B3}
        UnknownOpcode.of("26DA", "0x01,0xB3");
    }
```

The best fix I've found for this is to do something like this:

```java
(byte) Integer.decode(...)
```

Reply:
Did it as Short.decode().byteValue()